pallets/staking/src/weights.rs

```diff
@@ -77,6 +77,9 @@ pub trait WeightInfo {
     fn reap_stash(s: u32) -> Weight;
     fn new_era(v: u32, d: u32) -> Weight;
     fn npow_mint() -> Weight;
+    fn add_account_to_blacklist() -> Weight;
```

Comment:
The funciton of weight.rs  is auto created by benchmarking.rs .  see benchmarking.rs  for details